.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
add the following lines and see if anything changes:
```
--color
--format documentation
```

Reply:
Thank you so much Apu. Now I can see the result verbose and colourful output for test results.